packages/fast-development-site-react/src/components/site/category.tsx

```diff
@@ -6,6 +6,13 @@ export interface ISiteCategoryProps {
     name: string;
     component?: any;
     schema?: any;
+    status?: Status;
+}
+
+export enum Status {
```

Comment:
@eljefe223 Can you add documentation for each of these status' in the readme. We should likely also reference it in the readme for fast-components-react-msft and base since these status' will correlate directly to the components.

Reply:
Added to #8